Libplanet/Net/Swarm.cs

```diff
@@ -75,7 +76,7 @@ public partial class Swarm : ICollection<Peer>, IDisposable
         public Swarm(
             PrivateKey privateKey,
             TimeSpan dialTimeout,
-            IPAddress ipAddress = null,
+            string host = null,
             int? listenPort = null,
```

Comment:
Is it okay to pass only `host` and omit `listenPort`?

Reply:
Yes, in that case, it binds arbitrary ports through `_router.BindRandomPort()`.